src/goto-harness/recursive_initialization.cpp

```diff
@@ -428,6 +428,9 @@ std::string recursive_initialization_configt::to_string() const
 
 recursive_initializationt::array_convertert
 recursive_initializationt::default_array_member_initialization()
+irep_idt recursive_initializationt::get_fresh_global_name(
+  const std::string &symbol_name,
```

Comment:
`get_fresh_global_name` and `get_fresh_global_symexpr` look very similar. Are we sure the bulk of the work performed in one function, and then have the helpers call into it?